### PR TITLE
Skip problematic test on windows

### LINKED ISSUE #296: Unit tests on Windows run forever
And if I cancel them, they have e.g. the following error: (#289). Right now, we have more failed CI runs in our master than successful ones! Can someone relate the problems on Windows to some recent changes in the data base code? 
```
Traceback (most recent call last):
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\database\generic.py", line 578, in get_item_by_id
    return self.__get_items("id", int(item_id))[-1]
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Miniconda\envs\test\lib\multiprocessing\process.py", line 315, in _bootstrap
    self.run()
  File "C:\Miniconda\envs\test\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\job\generic.py", line 1663, in multiprocess_wrapper
    job_wrap = JobWrapper(
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\job\wrapper.py", line 50, in __init__
    self.job = pr.load(int(job_id))
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\project\generic.py", line 757, in load
    return self.load_from_jobpath(
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\project\generic.py", line 777, in load_from_jobpath
    job = jobpath(db=self.db, job_id=job_id, user=self.user)
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\job\path.py", line 468, in __init__
    db_entry = db.get_item_by_id(job_id)
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\database\generic.py", line 585, in get_item_by_id
    raise IndexError(
IndexError: ('Error when trying to find elements by given Job ID: ', IndexError('list index out of range'))
Coverage.py warning: Module pyiron_atomistics was never imported. (module-not-imported)
Coverage.py warning: No data was collected. (no-data-collected)
EProcess Process-2:
Traceback (most recent call last):
  File "C:\Miniconda\envs\test\lib\site-packages\pyiron_base\database\generic.py", line 578, in get_item_by_id
    return self.__get_items("id", int(item_id))[-1]
IndexError: list index out of range
```

### PR DESCRIPTION
**Pull request by niklassiemer**
According to #306 the Windows problems have been introduced in #241. This PR omits the new test of #241 if the OS is Windows.

### CODE REVIEW

**Comment by niklassiemer** [comment]
This cures the Windows - run forever mode! 🥳 
Remembering all the data base readout errors and seeing that this tests uses `Project.unpack()` to get one specific project to test on, I fear that the `unpack` method is able to corrupt our database - at least on windows!

File: .github/workflows/backwards.yml [hunk 1]
```diff
@@ -16,6 +16,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Merge Environments
+      shell: bash -l {0}
+      run: |
+        awk 'NR>3{ print $0 }' .ci_support/environment-notebooks.yml >> .ci_support/environment.yml
     - uses: conda-incubator/setup-miniconda@v2
```
Comment (jan-janssen) [inline .github/workflows/backwards.yml:23]: Is this change related to the windows tests?

**Comment by jan-janssen** [comment]
Is this just an `/` vs `\` issue in:

```
        to_dir = os.path.join(self.project.path, "..")
        shutil.copy("tests/static/lammps_test_files/get_structure_test.tar.gz", to_dir)
        shutil.copy("tests/static/lammps_test_files/export.csv", to_dir)
```

File: .github/workflows/backwards.yml (hunk 1 above)
Comment (niklassiemer) [reply .github/workflows/backwards.yml:23]: No, it is the same change as it was done for the notebook testing. In the tests the environment resolution took more than 30 minutes and resulted in lots of conflicts.

Comment (jan-janssen) [reply .github/workflows/backwards.yml:23]: Ok, right - now I see. Thanks for the explanation.

**Comment by niklassiemer** [comment]
Does the `shutil` not always work also with posix path? I actually do not know... @pmrv could you have a look and try to make the test running on windows as well?
